### PR TITLE
[ADD] hr_payroll_default_salary_structure

### PR DESCRIPTION
**Pull request by saltonmassally**
This addon allows you to define a default payroll structure for contracts

### CODE REVIEW

**Comment by ghost** [comment]
Please add support for multi company. If the config is multi-company, we should be able to set the field in the res.company form view.

**Comment by saltonmassally** [comment]
Will do; should I also remove setting the field from HR Settings or does that stay in

**Comment by ghost** [comment]
@tarzan0820 You may let the HR setting, it is the standard way for non-multi-company setup.

**Comment by feketemihai** [comment]
The same as in #176 , about the HR Settings, i think we should just leave the fields to be set on the company.
@dreispt ?

File: hr_payroll_default_salary_structure/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* Go to Settings > Human Resources and set your default structure
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/hr/issues/new?body=module:%20hr_employee_id%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (feketemihai) [inline hr_payroll_default_salary_structure/README.rst]: Please change the name of the module in link with the rigth one: hr_employee_id to hr_payroll_default_salary_structure.

File: hr_payroll_default_salary_structure/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    "name": "Default Salary Structure",
+    "version": "0.1",
```
Comment (feketemihai) [inline hr_payroll_default_salary_structure/__openerp__.py]: Please adjust version number to the new standards.

File: hr_payroll_default_salary_structure/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Configuration
+=============
+
+This module does not require any special configuration apart from enabling it as
+shown above.
+
+Usage
+=====
+* Go to Settings > Human Resources and set your default structure
+
```
Comment (feketemihai) [inline hr_payroll_default_salary_structure/README.rst:30]: Please add the Try on Runbot section.

**Comment by saltonmassally** [comment]
@feketemihai fixed

File: hr_payroll_default_salary_structure/README.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```
Comment (feketemihai) [inline hr_payroll_default_salary_structure/README.rst:33]: Please fix the repo_id and branch in link.

File: hr_payroll_default_salary_structure/README.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
```
Comment (feketemihai) [inline hr_payroll_default_salary_structure/README.rst:49]: Please fix link with the repo, module name and version.

**Comment by feketemihai** [comment]
@tarzan0820 @dufresnedavid Do somebody from you continue work on this, do you still need it for v8, there are too many modules on v8, which we don't have a continuation for a long time, so if nobody wants to continue them i think we should close it.

**Comment by feketemihai** [comment]
Please reopen the PR if it's important to you.

**Pull request closed without merging**